src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1744,10 +1782,16 @@ internal void ReportFieldContextualKeywordConflictIfAny(SyntaxNode syntax, Synta
             if (name == "field" &&
                 ContainingMember() is MethodSymbol { MethodKind: MethodKind.PropertyGet or MethodKind.PropertySet, AssociatedSymbol: PropertySymbol { IsIndexer: false } })
             {
-                var requiredVersion = MessageID.IDS_FeatureFieldKeyword.RequiredVersion();
-                diagnostics.Add(ErrorCode.INF_IdentifierConflictWithContextualKeyword, syntax, name, requiredVersion.ToDisplayString());
+                ReportFieldContextualKeywordConflict(syntax, identifier, diagnostics);
             }
         }
+
+        private static void ReportFieldContextualKeywordConflict(SyntaxNode syntax, SyntaxToken identifier, BindingDiagnosticBag diagnostics)
+        {
+            string name = identifier.Text;
+            var requiredVersion = MessageID.IDS_FeatureFieldKeyword.RequiredVersion();
+            diagnostics.Add(ErrorCode.INF_IdentifierConflictWithContextualKeyword, syntax, name, requiredVersion.ToDisplayString());
```

Comment:
>ErrorCode.INF_IdentifierConflictWithContextualKeyword [](http://example.com/codeflow?start=28&length=53)

Should be reported in `LanguageParser.ParseTermWithoutPostfix` rather than here.

Reply:
I was assuming the comment *is* the tracking. (It might be useful to move the diagnostic to the parser, but it doesn't seem necessary, so I didn't think it required an issue.)

Reply:
It's fine to not have an issue, but in that case I would expect a prototype comment, is there one present here that I missed?

Reply:
If we refine the diagnostic reporting to only report when `field` would not bind to a distinct symbol, reporting will need to stay in the `Binder`. Given that, I'll leave the code as is, with no PROTOTYPE comment for now.